cpp/tests/table/table_tests.cu

```diff
@@ -14,18 +14,138 @@
  * limitations under the License.
  */
 
+//#include <cudf/column/column.hpp>
+//#include <cudf/null_mask.hpp>
+//#include <cudf/types.hpp>
+//#include <cudf/utilities/type_dispatcher.hpp>
+//#include <tests/utilities/base_fixture.hpp>
+//#include <tests/utilities/column_utilities.hpp>
+//#include <tests/utilities/cudf_gtest.hpp>
+//#include <tests/utilities/type_list_utilities.hpp>
+//#include <tests/utilities/type_lists.hpp>
+
 #include <cudf/column/column.hpp>
 #include <cudf/column/column_view.hpp>
-#include <cudf/null_mask.hpp>
-#include <cudf/types.hpp>
-#include <cudf/utilities/type_dispatcher.hpp>
+#include <cudf/table/table.hpp>
+#include <cudf/table/table_view.hpp>
+
 #include <tests/utilities/base_fixture.hpp>
 #include <tests/utilities/column_utilities.hpp>
-#include <tests/utilities/cudf_gtest.hpp>
-#include <tests/utilities/type_list_utilities.hpp>
-#include <tests/utilities/type_lists.hpp>
+#include <tests/utilities/column_wrapper.hpp>
 
-#include <thrust/sequence.h>
 #include <random>
+#include <memory>
+
+template <typename T>
+using column_wrapper = cudf::test::fixed_width_column_wrapper<T>;
+
+using CVector     = std::vector<std::unique_ptr<cudf::column>>;
+using column      = cudf::column;
+using column_view = cudf::column_view;
+using TView       = cudf::table_view;
+using Table       = cudf::experimental::table;
+
+struct TableViewTest : public cudf::test::BaseFixture {};
+
+template <typename T>
+struct TableTest : public cudf::test::BaseFixture {
+  std::default_random_engine generator;
+  std::uniform_int_distribution<int> distribution{1000, 10000};
+  int random_size() { return distribution(generator); }
```

Comment:
If you inherit from the `UniformRandomGenerator` base class in `base_fixture.hpp` it handles the boilerplate for generating random numbers for you.

Reply:
I realized I do not need the random number generator in the test. Removed.